README.md

```diff
@@ -136,15 +136,15 @@ information on its usage.
 
```

Comment:
Content team (cc @arthurvr @agcolom @kswedberg who has helped reviewing past doc changes), please just let me know if you have thoughts for improvements. Thanks in advance.